libse/Utilities.cs

```diff
@@ -468,17 +468,20 @@ public static string AutoBreakLinePrivate(string text, int maximumLength, int me
                 var noTagLines = HtmlUtil.RemoveHtmlTags(text, true).SplitToLines();
                 if (noTagLines.Count == 2)
                 {
-                    var arr0 = noTagLines[0].Trim().TrimEnd('"', '\'').TrimEnd();
+                    const string LineCloseChars = ".?!)]♪";
+                    char[] quoteChars = { '"', '\'' };
+
+                    var arr0 = noTagLines[0].Trim().Trim(quoteChars).Trim();
                     if (language == "ar")
                     {
-                        if (arr0.EndsWith('-') && noTagLines[1].TrimStart().EndsWith('-') && arr0.Length > 1 && (".?!)]♪".Contains(arr0[0]) || arr0.StartsWith("--", StringComparison.Ordinal) || arr0.StartsWith('–')))
+                        if (arr0.EndsWith('-') && noTagLines[1].TrimEnd(quoteChars).EndsWith('-') && arr0.Length > 1 && (LineCloseChars.Contains(arr0[0]) || arr0.StartsWith("--", StringComparison.Ordinal) || arr0.StartsWith('–')))

```

Comment:
In arabic we want to trim from the end not start, since the hyphen will be placed from end of the line!